cnxepub/scripts/collated_single_html/main.py

```diff
@@ -34,9 +34,12 @@ def main(argv=None):
     parser.add_argument('-d', '--dump-tree', action='store_true',
                         help='Print out parsed model tree.')
 
-    parser.add_argument('-o', '--output', type=argparse.FileType('w'),
+    parser.add_argument('-o', '--output', type=argparse.FileType('w+'),
```

Comment:
cnx-epub-validate-collated -i data/TEAapphysics2-raw.epub -o - data/TEAapphysics2-baked.html >foo2.epubI just tested it. It worked.